docs/concepts/composition.md

```diff
@@ -161,813 +101,157 @@ spec:
-            Type:  object
-          Status:
-            Properties:
-              Address:
-                Description:  Address of this MySQL server.
-                Type:         string
-              Admin DSN:
-                Description:  DSN (Data Source Name) of the MySQL server.
-                Type:         string
-            Type:             object
-        Type:                 object
-Status:
-  Conditions:
-    Last Transition Time:  2020-05-15T05:30:45Z
-    Reason:                WatchingCompositeResource
-    Status:                True
-    Type:                  Established
-    Last Transition Time:  2020-05-15T05:30:45Z
-    Reason:                WatchingCompositeResourceClaim
-    Status:                True
-    Type:                  Offered
-  Controllers:
-    Composite Resource Claim Type:
-      API Version:  example.org/v1alpha1
-      Kind:         MySQLInstance
-    Composite Resource Type:
-      API Version:  example.org/v1alpha1
-      Kind:         CompositeMySQLInstance
-Events:
-  Type     Reason              Age                   From                                                             Message
-  ----     ------              ----                  ----                                                             -------
-  Normal   EstablishComposite  4m10s                 defined/compositeresourcedefinition.apiextensions.crossplane.io  waiting for composite resource CustomResourceDefinition to be established
-  Normal   OfferClaim          4m10s                 offered/compositeresourcedefinition.apiextensions.crossplane.io  waiting for composite resource claim CustomResourceDefinition to be established
-  Normal   ApplyClusterRoles   4m9s (x4 over 4m10s)  rbac/compositeresourcedefinition.apiextensions.crossplane.io     Applied RBAC ClusterRoles
-  Normal   RenderCRD           4m7s (x8 over 4m10s)  defined/compositeresourcedefinition.apiextensions.crossplane.io  Rendered composite resource CustomResourceDefinition
-  Normal   EstablishComposite  4m7s (x6 over 4m10s)  defined/compositeresourcedefinition.apiextensions.crossplane.io  Applied composite resource CustomResourceDefinition
-  Normal   EstablishComposite  4m7s (x5 over 4m10s)  defined/compositeresourcedefinition.apiextensions.crossplane.io  (Re)started composite resource controller
-  Normal   RenderCRD           4m7s (x6 over 4m10s)  offered/compositeresourcedefinition.apiextensions.crossplane.io  Rendered composite resource claim CustomResourceDefinition
-  Normal   OfferClaim          4m7s (x4 over 4m10s)  offered/compositeresourcedefinition.apiextensions.crossplane.io  Applied composite resource claim CustomResourceDefinition
-  Normal   OfferClaim          4m7s (x3 over 4m10s)  offered/compositeresourcedefinition.apiextensions.crossplane.io  (Re)started composite resource claim controller
 ```
 
-### Specify How Your Resource May Be Composed
+> You might notice that the `XPostgreSQLInstance` example above has some fields
```

Comment:
The content is great but it doesn't need to be a note.

Reply:
De-noted.